third_party/proxy/config/config.go

```diff
@@ -347,7 +347,8 @@ func (c *EndpointSliceConfig) handleUpdateEndpointSlice(oldObj, newObj interface
 		return
 	}
 	for _, h := range c.eventHandlers {
-		klog.V(4).Infof("Calling handler.OnEndpointSliceUpdate")
+		klog.V(4).Infof("Calling handler.OnEndpointSliceUpdate from %+v", oldEndpointSlice)
```

Comment:
Use klog.V(4).InfoS()? 
One log including both old and new Endpointslice is enought.

Reply:
Updated